### PR TITLE
Update go-build to v0.65.1

### PR DESCRIPTION
**Pull request by doublek**

### CODE REVIEW

**Review by sujeet-kr — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by robbrockbank — APPROVED** [review]

**Pull request merged**
